birdhouse/components/weaver/config/weaver/wps_processes.yml.template

```diff
@@ -0,0 +1,37 @@
+# Details:
+#   https://github.com/crim-ca/weaver/blob/master/config/wps_processes.yml.example
+#
+# Using 'processes' section:
+#   For each of the below WPS service providers, Weaver would iterate over listed WPS processes returned by their
+#   GetCapabilities response and would register them as individual WPS-REST processes. Each process will be defined
+#   as a local snapshot of the process at that point in time. They would be static an not consider process changes.
+# Using 'providers' section:
+#   Weaver will register each service reference directly without fetching every underlying process immediately.
+#   Later calling process description or execution will make Weaver obtain that definition dynamically from the
+#   referenced WPS DescribeProcess/Execute response to provide a live response.
+
+# Note:  
+#   All service providers and underlying processes must be publicly accessible for GetCapabilities and DescribeProcess
+#   at boot time in order to allow Weaver to retrieve the information. The application will skip unresolvable references
+#   including unauthorized and forbidden access.
+
+# processes:    # static local snapshot
+providers:      # dynamic remote reference
+  - name: catalog
```

Comment:
Oh, so all the providers are already listed here.  So why need them to have anon read access in magpie again?  Sorry this might be a dumb question since I am new to weaver.

Reply:
If there is a new bird, can it hook itself in to weaver like it can hook itself in to magpie right now?

Reply:
Before registering the providers, weaver tries to do a GetCapabilities to validate the endpoint.